### PR TITLE
feat: more v13 changes

### PR DESCRIPTION
**Pull request by monbrey**
Just another update to the v13 changes page that weren't included when the PR was merged.

### CODE REVIEW

File: guide/additional-info/changes-in-v13.md [hunk 1]
```diff
@@ -694,10 +766,31 @@ Shortcuts to Util methods which were previously exported at the top level have b
...
+
 #### Util#resolveString
 
 The `Util#resolveString` method has been removed. discord.js now enforces that users provide strings where expected rather than resolving one on their behalf.
 
+### VoiceState
+
+#### VoiceState#kick
+
+The `VoiceState#kick` method has been renamed to `VoiceState#disconnect`
```
Comment (J-Human) [inline guide/additional-info/changes-in-v13.md:781]:
```suggestion
The `VoiceState#kick` method has been renamed to `VoiceState#disconnect`.
```

**Review by J-Human — CHANGES_REQUESTED** [review]

File: guide/additional-info/changes-in-v13.md [hunk 2]
```diff
@@ -445,6 +495,18 @@ This method has been removed, with functionality replaced by the new `Permission
...
+
+#### GuildChannel#fetchInvites
+
+These methods have been removed from `GuildChannel` and placed only on subclasses for which invites can be created. These are `TextChannel`, `NewsChannel`, `VoiceChannel`, `StageChannel` and `StoreChannel`.
+
+On these subclasses, the method now supports additional options:
+
+- `targetUser` to target the invite to join a particular streaming user
+- `targetApplication` to target the invite to a particular Discord activity
+- `targetType`
```
Comment (kyranet) [inline guide/additional-info/changes-in-v13.md:508]: This should probably be explained, like the other two fields are.

File: guide/additional-info/changes-in-v13.md [hunk 3]
```diff
@@ -694,10 +766,31 @@ Shortcuts to Util methods which were previously exported at the top level have b
...
+
+### WebhookClient
+
+The `WebhookClient` constructor no longer accepts `id, token` as the first two parameters, instead taking a `data` object. This object supports an additional option `url`, allowing creation of a `WebhookClient` from a webhook URL.
+
+```diff
+- new WebhookClient(id, token, options);
++ new WebhookClient({ id, token }, options);
+
++ new WebhookClient{ url }, options);
```
Comment (kyranet) [inline guide/additional-info/changes-in-v13.md:791]:
```suggestion
+ new WebhookClient({ url }, options);
```

Also a bit weird to see it as two objects, but oh well, a bit late for that.

**Review by kyranet — CHANGES_REQUESTED** [review]

File: guide/additional-info/changes-in-v13.md (hunk 2 above)
Comment (J-Human) [reply guide/additional-info/changes-in-v13.md:508]: It possibly could be "a type that grants additional features to the invite, 1 for a streaming user, 2 for voice channel activities..."?

Comment (monbrey) [reply guide/additional-info/changes-in-v13.md:508]: I mean isn't the type just specifying which of the above two it is?

Comment (kyranet) [reply guide/additional-info/changes-in-v13.md:508]: Then it should clarify that by describing it.

File: guide/additional-info/changes-in-v13.md [hunk 4]
```diff
@@ -458,6 +520,10 @@ This method has been removed, with functionality replaced by the new `Permission
 
 This method no longer returns a Collection of PermissionOverwrites, instead providing access to the `PermissionOverwriteManager`.
 
+#### GuildChannel#setTopic
+
+The `GuildChannel#setTopic` method has been removed and placed only on subclasses for which topics can be set. These are `TextChannel`, `NewsChannel` and `StageChannel`.
```
Comment (Danktuary) [inline guide/additional-info/changes-in-v13.md:525]:
```suggestion
The `GuildChannel#setTopic` method has been removed and placed only on subclasses for which topics can be set. These are `TextChannel`, `NewsChannel`, and `StageChannel`.
```

**Comment by noftaly** [comment]
I cannot make a review outside of the diff, but it would be great to add `GuildMember#lastMessageChannelId` alongside `GuildMember#lastMessage`/`GuildMember#lastMessageId` :)

File: guide/additional-info/changes-in-v13.md (hunk 2 above)
Comment (monbrey) [inline guide/additional-info/changes-in-v13.md:508]:
```suggestion
- `targetType` defines the type of the target for this invite; user or application
```

**Review by kyranet — APPROVED** [review]

**Pull request merged**